Tasks/CopyFilesToMachines/CopyFilesHelper.ps1

```diff
@@ -1,4 +1,4 @@
-function CompleteResourceOperation
+function Complete-ResourceOperations
```

Comment:
Why this change. I thought this function still deals with only one operation and not multiple operations.